binding/Binding/GRContext.cs

```diff
@@ -47,7 +47,21 @@ public static GRContext Create (GRBackend backend, IntPtr backendContext)
 				case GRBackend.OpenGL:
 					return GetObject<GRContext> (SkiaApi.gr_context_make_gl (backendContext));
 				case GRBackend.Vulkan:
+					return GetObject<GRContext> (SkiaApi.gr_context_make_vulkan (backendContext));
+				default:
+					throw new ArgumentOutOfRangeException (nameof (backend));
+			}
+		}
+
+		public static GRContext Create (GRBackend backend, GRVkBackendContext backendContext)
+		{
+			switch (backend) {
```

Comment:
I think we can improve this a fair bit by not using a switch.

Because this method only supports Vulkan, we might be better served with a:

```csharp
if (backend != GRBackend.Vulkan)
    throw new ArgumentOutOfRangeException (nameof (backend));
```

And we can probably do this for the other create methods as well. This will also help future-proof that enum.

Reply:
I guess it doesn't make sense to touch the obsolete one:

```
		[Obsolete ("Use Create(GRBackend, GRGlInterface) instead.")]
		public static GRContext Create (GRBackend backend, IntPtr backendContext)
```

This overload is also tricky, because I cannot easily implement anything that makes sense for Vulkan:
```
		public static GRContext Create (GRBackend backend)
```

Reply:
We can just throw for Vulkan, and the exception message can say to use the overload.